tools/analysis_tools/benchmark.py

```diff
@@ -33,16 +36,20 @@ def parse_args():
         'It also allows nested list/tuple values, e.g. key="[(a,b),(c,d)]" '
         'Note that the quotation marks are necessary and that no white space '
         'is allowed.')
+    parser.add_argument(
+        '--launcher',
+        choices=['none', 'pytorch', 'slurm', 'mpi'],
+        default='none',
+        help='job launcher')
+    parser.add_argument('--local_rank', type=int, default=0)
     args = parser.parse_args()
+    if 'LOCAL_RANK' not in os.environ:
+        os.environ['LOCAL_RANK'] = str(args.local_rank)
     return args
 
 
-def main():
-    args = parse_args()
-
-    cfg = Config.fromfile(args.config)
-    if args.cfg_options is not None:
-        cfg.merge_from_dict(args.cfg_options)
+def calc_inference_fps(cfg, checkpoint, max_iter, log_interval,
+                       is_fuse_conv_bn):
     # import modules from string list.
     if cfg.get('custom_imports', None):
```

Comment:
We can delete lines 54-56 because MMCV already do that part https://github.com/open-mmlab/mmcv/blob/master/mmcv/utils/config.py#L256